src/com/gh4a/fragment/CommentBoxFragment.java

```diff
@@ -63,6 +68,10 @@ public void addQuote(CharSequence text) {
         UiUtils.showImeForView(mCommentEditor);
     }
 
+    public void setMentionUsers(List<User> suggestions) {
+        mAdapter.replace(suggestions);
```

Comment:
Given this is called from a Loader callback, is mAdapter guaranteed to be != null here?